src/arguments/utility.py

```diff
@@ -75,57 +77,102 @@ def get_ans(self, questions_list):
         This function prints the answer to the queries
         (question and tags) provided by the user. It does so
         in the following manner :
-
-        1) Takes in account only th first question id from the
-        list of question ids
-
-        2) Tries to get a response from the url obtained by appending
-        question id to the search_content_url variable
-
-        3) Use the data received from the above request and loop
-        through it to print the answer
+        1) Gets the details of all the relavant question and stores their title, link and id in "question_data" list. [See line 104]
```

Comment:
What does this mean `[See line 104]`? Because at #L104 , `# Clear terminal` is written

Reply:
That was a wrong line number. I'll remove it, its unnecessary.